src/features/chat-window/chat.module.scss

```diff
@@ -18,23 +18,23 @@
   }
 
   &Main {
-    border: 1px solid #eee;
     bottom: 0;
     display: flex;
+    height: 100vh;
```

Comment:
Don't use height with `vh` unit. It will always have problems on mobile devices.